cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -255,6 +255,15 @@ def verify():
     wait_for(verify, message="email receive check", delay=5)
 
 
+@pytest.mark.meta(
+    blockers=[
+        BZ(
+            1662638,
+            forced_streams=["5.9", "5.10", "upstream"],
+            unblock=lambda provider: not provider.key == "vsphere65-nested",
```

Comment:
After comparison negation? I would go in simple way  like  `provider.key != "vsphere65-nested"`

Reply:
@digitronik is this formatting ok? I haven't seen such in our framework, what do you think?

Reply:
@lina-nikiforova I've formatted it with black.

Reply:
@lina-nikiforova: yup, we accepted `black` as code formatter.